examples/paddle/mnist/README.md

```diff
@@ -0,0 +1,6 @@
+# Basic MNIST Example (BF16)

```

Comment:
Could we add an example of FP8 usage in the same script as well?

Reply:
Yes. It will come later in the next PR.

Reply:
I hope @Tom-Zheng to think FP8 support more systematically when adding FP8 support. Thus, this PR focuses on BF16 only, then next PR will update everything about FP8.